types/carbon__elements/index.d.ts

```diff
@@ -0,0 +1,6458 @@
+// Type definitions for @carbon/elements 10.24
+// Project: https://github.com/carbon-design-system/carbon/tree/master/packages/elements
+// Definitions by: Eric Liu <https://github.com/metonym>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.1
```

Comment:
this is month old release,
is there a specific reason to use 4.1 only?
https://github.com/microsoft/TypeScript/releases/tag/v4.1.2

Reply:
No reason in particular. Should I downgrade to 4.0?

Reply:
@metonym not sure, other @carbon/* stuff seems to have 4.0, 3.8 or no versioning at all.
Let's keep it 'as is' for now. Usually that versioning is due to specific language features inroduced
with e.g. 4.1 version (like named tuples, or something similar).